docs/guide/saving-and-retrieving-data.md

```diff
@@ -46,7 +46,7 @@ $ ember generate controller posts
 
 ```js
 // app/controllers/posts.js
-export default Ember.ArrayController.extend({
+export default Ember.Controller.extend({
```

Comment:
It looks like Ember.Controller does not support the ```sortProperties``` or ```sortAscending``` attributes (https://www.emberjs.com/api/classes/Ember.Controller.html). There's more information about the deprecation here https://emberjs.com/deprecations/v1.x/#toc_sortablemixin. Looks like you can swap it out with computed methods like so:

```javascript
export default Ember.Controller.extend({
  sortProperties: Ember.computed.sortBy('model', 'sortModelBy'),
  sortModelBy: ['timestamp:desc'],
  actions: {
    ...
  }
});